### PR TITLE
[sdk-nightly] - use only sanity playbook without integrations

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9329)

## Description
* Run the SDK nightly only against sanity test which do not require any integration or any external api source.
* Skipping `xpanse` because there is no reason to collect tests for it in the SDK-nightly **ATM**

[Successful nightly](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/746789)

### CODE REVIEW

**Review by JudahSchwartz — APPROVED** [review]
check that really expanse is not needed

File: Tests/scripts/collect_tests/collect_tests.py [hunk 1]
```diff
@@ -1408,7 +1408,20 @@ def get_e2e_packs(self) -> set[str]:
 class SDKNightlyTestCollector(TestCollector):
 
     def _collect(self) -> CollectionResult | None:
-        return self.sanity_tests
+        if self.marketplace == MarketplaceVersions.XPANSE:
+            return None
```
[the remark this answers is not in the document]
Comment (GuyAfik) [reply Tests/scripts/collect_tests/collect_tests.py:1412]: We don't need to collect any tests for xpanse (this is redundant) as we do not run any tests against any xpanse instance

**Review by MLainer1 — APPROVED** [review]
🍪

File: Tests/scripts/collect_tests/collect_tests.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dorschw) [reply Tests/scripts/collect_tests/collect_tests.py:1412]: the condition looks good, but why move the test out of `self.sanity_tests`?

[the remark this answers is not in the document]
Comment (GuyAfik) [reply Tests/scripts/collect_tests/collect_tests.py:1412]: np, will be done

[the remark this answers is not in the document]
Comment (GuyAfik) [reply Tests/scripts/collect_tests/collect_tests.py]: done

**Pull request merged**